docs/changelog.md

```diff
@@ -275,6 +275,18 @@ what we publish.
   trailing whitespaces. ([PR #2683](https://github.com/modularml/mojo/pull/2683)
   by [@fknfilewalker](https://github.com/fknfilewalker))
 
+- `List` now has a `delete_batch_of_indexes` method that provides a way to delete multiple elements by index in a batch.
+
+  - Even if provided twice, it won't delete two times the same index.
+  - The order of removal is reverse of sorted provided indexes.
+
+  ```mojo
+  x = List[Int](0,10,20,30,40)
+  x.delete_batch_of_indexes(0,4)
```

Comment:
Since we lack the language feature to remove subranges via the `del` keyword, I propose we implement the `__delitem__` member function and call it directly until we land the language feature enabling the use `del`.  @JoeLoser Thoughts?

Reply:
Hello, 

new commit changed the method name to `__delitem__` and includes an overload that takes only one position as an integer.

That way, it won't create a set and sort an array if there is only one index. ( ```_ = self.pop(position)```)

- Should there be any debug_asserts that asserts that `len(self) > index` and `index >= 0` ?

- Should the methods just don't do ```self.pop(position)``` if position is out of bound ?